source/nodejs/adaptivecards-designer/src/card-designer.ts

```diff
@@ -350,16 +364,22 @@ export class CardDesigner {
                 errorPane.classList.add("acd-hidden");
             }
         };
+        this._designerSurface.onStartDrag = (sender: Designer.CardDesignerSurface) => {
+            this._startDragPayload = JSON.parse(this.getCurrentCardEditorPayload());
+        };
+        this._designerSurface.onEndDrag = (sender: Designer.CardDesignerSurface, wasCancelled: boolean) => {
+            if (wasCancelled) {
+                this.setCardPayload(this._startDragPayload, false);
```

Comment:
nit: is this statement hit when wasCancelled is set true?
nit: adding a hint of comments would be helpful.

Reply:
This is an event handler. It will be called with wasCancelled either set to true or false.

Reply:
i was asking this to make sure when "wasCancelled"" is set true, it means it's cancelled; if not, having a comment woudl be helpful; just a suggestion...

---
In reply to: [333249001](https://github.com/microsoft/AdaptiveCards/pull/3508#discussion_r333249001) [](ancestors = 333249001)

Reply:
Seems pretty self explanatory to me.. What else could `wasCancelle` mean in an `endDrag` event handler? Comment would state what is already quite obvious IMO.